src/ResourceManager/DataLakeStore/AzureRM.DataLakeStore.psd1

```diff
@@ -152,14 +151,45 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* NOTE: This is a breaking change release. Please see the migration guide (https://aka.ms/azps-migration-guide) for a full list of breaking changes introduced.
-* Removed the Obsolete Properties fields in PSDataLakeStoreAccount.cs and its assoicated files
-* Changed one of the two OutputTypes of Get-AzureRmDataLakeStoreAccount
-    - List\<PSDataLakeStoreAccount> to List\<PSDataLakeStoreAccountBasic>
-    - The properties of PSDataLakeStoreAccountBasic is a strict subset of the properties of PSDataLakeStoreAccount
-    - The additional properties that are in PSDataLakeStoreAccount are not returned by the service.  Therefore, this change is to reflect this accurately. These additional properties are still in PSDataLakeStoreAccountBasic, but they are tagged as Obsolete.
-* Add support for online help
-    - Run Get-Help with the -Online parameter to open the online help in your default Internet browser'
+        ReleaseNotes = '
+		* Moved from AutoRest sdk to our version of Dataplane sdk for Azure data lake.
```

Comment:
This should be customer-relevant information.  You should specify which cmdlets have changed, and either have a link to the breaking changes document here (if there are many changes), or specify the specific changes (if there are only a few).

Details like the folder structure and the technology used for thedata plane client are irrelevant to customers.

Also, please make sure this information is included in your changelog.

Reply:
@markcowl Fixed it and put the same contents under ChangeLog.